src/isPalindrome.js

```diff
@@ -19,7 +19,22 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  if (str === '') {
+    return true;
+  }
+
+  const lowerCaseStr = str.toLowerCase();
+  const lettersRegEx = /\w/g;
```

Comment:
он один в этой функции, можно не уточнять
```suggestion
  const pattern = /\w/g;
```